# Stop the robot
---
bool success
string msg
